README.md

```diff
@@ -33,7 +33,7 @@ For details of the other Kata Containers repositories, see the
 (CLI) part of the Kata Containers runtime component. It leverages the
 [virtcontainers](virtcontainers)
 package to provide a high-performance standards-compliant runtime that creates
-hardware-virtualized containers.
+hardware-virtualized [Linux*](https://www.kernel.org/) containers running on Linux hosts.
```

Comment:
Is there a reason for the `*` in the hyperlink? :)

(Not strictly related, that sentence it's hard to read.)

Reply:
Our doc folk suggest it, yes, when first introducing a project/product/company name.

Reply:
Normally it has a correlating footnote about 'other names and brands' etc. See https://clearlinux.org/ for an example....
So, I guess the question is:
 - do we also need footnotes
 - or can we drop the `*`s. I see for instance there are none on https://katacontainers.io/

I'm going to rope in @ClaireMassey  and @klynnrif to see if we can get a decision :-)

Reply:
I don’t think it’s strictly required in this instance (this is usually a corporate CYA thing?)? I’d defer to OSF folks here. @ttx?

Reply:
I think you are right @egernst - I've zapped it ;)

Reply:
Yes I don't think we need any "other brands" CYA footnote in the open source project documentation -- at least I haven't seen it anywhere in OpenStack documentation.

Reply:
@jodh-intel I agree with @egernst as well (despite being late to the party) - I believe it's a corporate thing. It doesn't appear in the writing documentation/style guide for OpenStack.

Reply:
Thanks @klynnrif.